#!/usr/bin/env python

PACKAGE='motion_pro'
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("start", bool_t, 0, "start", False)
gen.add("robot_id", int_t, 0, "robot id", 0, 0, 11)

start = gen.add_group("start_point", state = True)
start.add("start_x", double_t, 0, "x",0, -9, 9)
start.add("start_y", double_t, 0, "y",-3, -7, 7)

end = gen.add_group("end_point", state = True)
end.add("end_x", double_t, 0, "x", -4, -9, 9)
end.add("end_y", double_t, 0, "y", -3, -7, 7)

gen.add("dist_step", int_t, 0, "dist step", 3, 1, 100)
gen.add("ang_step", int_t, 0, "angle step", 8, 1, 100)
gen.add("init_phase", double_t, 0, "init phase", 0, 0, 360)

exit(gen.generate(PACKAGE, "motion_pro", "motion_pro"))

